style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+#header-section {
+  width: 100%;
+  height: 104px;
+  background-image: url('images/logo-image.svg');
+  background-repeat: no-repeat;
+  background-position: top left;
+  background-size: contain;
+}
+
+#header-section div {
+  display: none;
+  float: left;
+  height: 74px;
+  width: 74px;
+}
+
+#header-section nav a {
+  float: right;
+}
+
+#section-main {
+  width: 100vw;
+  display: block;
+  padding: 0 12px;
+}
+
+#section-main h1 {
+  font-family: 400;
+  font-size: 34px;
+  color: #1a2236;
+  font-style: regular;
+  position: absolute;
+  top: 120px;
+  max-width: 800px;
+  height: 110px;
+  text-align: start;
+}
+
+#section-main p {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  color: #1a2236;
+  font-style: regular;
+  position: absolute;
+  top: 208px;
+  left: 16px;
+  text-align: start;
+}
+
+#second-main {
+  width: 100%;
+  height: 50px;
+  position: absolute;
+  top: 576px;
+  left: 16px;
+  background-color: #1a2236;
+  border: 1px solid tomato;
+}
+
+#second-main button {
+  color: #fff;
+  align-items: center;
+  height: 26.47px;
+  background-color: #1a2236;
+  border: none;
+  position: absolute;
+  top: -6px;
+  padding: 10px 40%;
+  margin-top: 10px;
+}
```

Comment:
- [ ] It is also part of the requirement to implement the button interactions (enable, hover etc). Currently, your button has no effects.

![image](https://user-images.githubusercontent.com/29071984/170427075-f2ca6879-6ef0-4a61-8d59-5f8da1125ac1.png)

Please implement the following button interaction in your project
![image](https://user-images.githubusercontent.com/29071984/170427321-698efbe7-e084-4ef2-93a3-a6b3f6430fc7.png)